experimental/dds/tree2/src/test/utils.ts

```diff
@@ -496,6 +499,30 @@ export function validateTree(tree: ISharedTreeView, expected: JsonableTree[]): v
 	assert.deepEqual(actual, expected);
 }
 
+export function makeTreeFromJson(json: JsonCompatible[] | JsonCompatible): ISharedTree {
+	const cursors = Array.isArray(json) ? json.map(singleJsonCursor) : singleJsonCursor(json);
+	return makeTreeFromCursor(cursors, jsonSchema);
+}
+
+export function makeTreeFromCursor(
+	cursor: ITreeCursorSynchronous[] | ITreeCursorSynchronous,
+	schemaData?: SchemaData,
```

Comment:
if schema isn't provided, the resulting tree will be out of schema, which is invalid.

Also if you don't really require input as cursors, refactoring this to just wrap tree.schematize to load the initial schema and tree content would be nice. It would let you provide a strongly typed schema aware contextual node data for the initial tree as well as reusing our initialization logic

Reply:
Apparently support for making branches without needing an actual tree has landed, so maybe use that instead?